data/search.lua

```diff
@@ -192,6 +198,8 @@ function search:Add(item)
     search:addStringToIndex(self.indicies.equipmentlocation, _G[item.itemInfo.itemEquipLoc], item.slotkey)
   end
 
+  search:addStringToIndex(self.indicies.binding, item.bindingInfo.binding, item.slotkey)
```

Comment:
because you will want to move this to an enum, you'll probably also want a map of enum -> string values in constants.lua and index those strings.